### ISSUE TITLE
Accessibility: Screen reader doesn't announce whether each of the feature selection dropdowns is expanded/collapsed

### ISSUE THREAD

**Issue by bazizi** [wixtoolset/issues#6389]
## Bugs

If this issue is a bug:

* Which version of WiX are you building with?
> (WiX version in major.minor.release.build format)

The version of *wix.dll* used is *3.9.2207*
![image](https://user-images.githubusercontent.com/7786414/110671121-06d12200-8183-11eb-8764-dc63d3452a48.png)
---
* Which version of Visual Studio are you building with (if any)?
> (Visual Studio version, including year and update)

We don't use Visual Studio. We use MSBuild version 16.4.0.56107.
---
* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?
> (WiX Toolset Visual Studio Extension version in major.minor.release.build format)

N/A
---
* Which version of .NET are you building with?
> (.NET version)

3.5
---
* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?
> (Windows version)

Windows 10 Version 20H2 (OS Build 19042.804)
![image](https://user-images.githubusercontent.com/7786414/110672179-303e7d80-8184-11eb-85cc-6916e06bb334.png)
---
* Describe the problem and the steps to reproduce it.
> (Be as specific as you can and err on the side of providing too much information, including code, error messages, command lines you used to invoke the build, and so forth.)

This is an accessibility issue. To reproduce, you need to have NVDA (2019.2.1) screenreader installed:
https://www.nvaccess.org/download/

## Repro steps:
- Select and run msoledbsql.msi file. The installer can be downloaded from here: https://docs.microsoft.com/en-us/sql/connect/oledb/download-oledb-driver-for-sql-server?view=sql-server-ver15#download
- "Welcome to the OLE DB Driver for SQL server Setup" screen will open.
- Select and press "Next" button, License agreement screen should open.
- Select "I accept the terms in license agreement" radio button and press next button.
- Feature Selection screen should open.
- Start screenreader and navigate the SelectionTree UI using down arrow key to the OLEDB driver dropdown and check whether its announce the expand/collapsed state or not.
---
* Describe the behavior you expected and how it differed from the actual behavior.
> (Be as specific as you can and err on the side of providing too much information.)

The following video provided by the accessibility engineer explains the issue and repro steps:
https://user-images.githubusercontent.com/7786414/110692246-03e22b80-819b-11eb-9c10-06b56a9995ee.mp4

## Expected
Screenreader should notify the user about the expanded/collapsed state of the feature selection dropdown.

## Actual
Screenreader will not announce the expanded/collapsed state of the feature selection dropdown. This is likely an accessibility issue with the [Control element of type SelectionTree](https://wixtoolset.org/documentation/manual/v3/xsd/wix/control.html) in WiX. It seems that internally context menus are used instead of drop downs for such elements.

**Comment by barnson** [comment]
This package isn't using the WixUI dialog library. If Windows Installer doesn't support the accessibility behavior you're looking for, there's nothing WiX can do to add it.
